core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -115,7 +115,7 @@ public String getPath() {
   }
 
   /**
-   * @return the file UFS path
+   * @return the file UFS (Under File System) path
```

Comment:
Please. define a summary for the javadoc (no problem if it is a little bit redundant)

Reply:
According to the [new Alluxio convention](https://github.com/Alluxio/alluxio/pull/11797/files), getters and setters should omit the method description if it is redundant and only use `@param` and `@return` descriptions. This is recent.